pkg/bootstrap/config/types.go

```diff
@@ -16,7 +16,20 @@
 
 package config
 
-import "github.com/IOTechSystems/go-mod-edge-utils/pkg/mqtt5/models"
+import (
+	"fmt"
+
+	"github.com/IOTechSystems/go-mod-edge-utils/pkg/mqtt5/models"
+	"github.com/IOTechSystems/go-mod-edge-utils/pkg/secrets/client"
+	"github.com/IOTechSystems/go-mod-edge-utils/pkg/secrets/types"
+)
+
+const (
+	ServiceTypeApp = "app-service"
+	//TOOD: add security-service to use in place of useSecretProvider
```

Comment:
app-service constants is not required

Reply:
This constant is used in SecureProvider.GetSelfJWT
https://github.com/edgexfoundry/go-mod-bootstrap/blob/cfc1a99971599b4a4df9b3195e5ae74953b767d2/bootstrap/secret/secure.go#L445-L449
https://github.com/edgexfoundry/go-mod-bootstrap/blob/cfc1a99971599b4a4df9b3195e5ae74953b767d2/bootstrap/secret/secure.go#L488-L495

Reply:
this is no for edgex, so overwriteAppServiceKey related logic should also be removed

Reply:
removed